analysis.js

```diff
@@ -2,70 +2,166 @@ $("#chart-weeklyDatePicker").datetimepicker({
   format: 'YYYY-MM-DD'
 });
 
-$("#chart-weeklyDatePicker").on("dp.change", function() {
-  var value = $("#chart-weeklyDatePicker").val();
-  var firstDate = moment(value, "YYYY-MM-DD").day(0).format("YYYY-MM-DD");
-  var lastDate = moment(value, "YYYY-MM-DD").day(6).format("YYYY-MM-DD");
-
-  $('#analysis-name').html('<h3 class="text-center">資料載入中...</h3>');
-  $("#chart-weeklyDatePicker").val(firstDate + "~" + lastDate);
+$("#chart-weeklyDatePickerStart").datetimepicker({
+  format: 'YYYY-MM-DD'
+});
 
-  window.fetchWeek($("#chart-weeklyDatePicker").val(), function() {
-    updateTownFormAndTitle();
-  });
+$("#chart-weeklyDatePickerEnd").datetimepicker({
+  format: 'YYYY-MM-DD'
 });
 
+function compareDate(firstdate, enddate, start, end)
```

Comment:
The function is never called. Because there are `compareDate` functions exists in `table.js` and `app.js`, the identifies conflicts.